src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -201,6 +201,15 @@ public DebuggedProcess(bool bLaunched, LaunchOptions launchOptions, ISampleEngin
                 {
                     this.Init(localTransport, _launchOptions);
                 }
+
+                // Only need to know the debugger pid on Linux and OS X local launch to detect whether
+                // the debugger is closed. If the debugger is not running anymore, the response (^exit)
+                // to the -gdb-exit command is faked to allow MIEngine to shut down.
+                _localDebuggerPid = localTransport.DebuggerPid;
+
+                // We are launching the debugger in the background, so the console will have
+                // "[job number] pid". So we clear that.
+                ResetConsole();
```

Comment:
This is currently not going to work for clrdbg - we don't support the shell command or the console exec command. Can we clear the console in the bash command instead?